config.dist.php

```diff
@@ -63,28 +63,30 @@
 ];
 
 // Define other looking glasses, this is useful if you have multiple networks and looking glasses;
-const LG_LOCATIONS = [
+/* const LG_LOCATIONS = [
     'Location A' => 'https://github.com/hybula/lookingglass/',
     'Location B' => 'https://github.com/hybula/lookingglass/',
     'Location C' => 'https://github.com/hybula/lookingglass/',
-];
+]; */
+const LG_LOCATIONS = [];
 
 // Enable the iPerf info inside the speedtest block, set to false to disable;
 const LG_SPEEDTEST_IPERF = true;
 // Define the label of an incoming iPerf test;
 const LG_SPEEDTEST_LABEL_INCOMING = 'iPerf3 Incoming';
 // Define the command to use to test incoming speed using iPerf, preferable iPerf3;
-const LG_SPEEDTEST_CMD_INCOMING = 'iperf3 -4 -c hostname -p 5201 -P 4';
+const LG_SPEEDTEST_CMD_INCOMING = 'iperf3 -4 -c ' . LG_IPV4 . ' -p 5201 -P 4';
```

Comment:
This should not change, because not everybody runs the iperf server on the looking glass server.

Reply:
I didn't think of this scenario at all. Sorry, I'll change these back and other corresponding parts as well.